endpoints.json

```diff
@@ -25,5 +25,21 @@
         }
       ]
     }
+  },
+  "GET /api/articles/:article_id": {
+    "description": "serves up the chosen article via its ID",
+    "queries": ["article_id"],
```

Comment:
the article id is a param on the endpoint - a query would be anyhting following the ? in an endpoint so article id isn't a query :)